hepdata/cli.py

```diff
@@ -416,3 +417,21 @@ def reindex():
 def write_stats_to_files():
     """Writes some statistics on number of submissions per Coordinator to files."""
     write_submissions_to_files()
+
+
+@cli.command()
+@with_appcontext
+@click.option('--inspire-id', '-i', type=str, required=True, help='Specify inspire ID of record to update.')
+@click.option('--recid', '-r', type=str, default='', help='Specify an HEPData record ID to update given the Inspire ID. This is to be used for not yet finilised submissions.')
+@click.option('--send_email', '-e', default=False, type=bool, help='Whether or not to send email about update.')
+def cli_update_record_info(inspire_id, recid='', send_email=False):
+    update_record_info(inspire_id, recid, send_email)
+
+
+@cli.command()
+@with_appcontext
+@click.option('--date', '-d', type=str, required=True, help='Specify date since when to update records.')
+def update_records_info_since(date):
+    inspire_ids = get_all_updated_records_since_date(date, verbose=True)
+    for inspire_id in inspire_ids:
+        update_record_info(inspire_id)
```

Comment:
I don't see `update_all_records_info()` in the latest commit?  And probably these commands should appear inside a group:
```python
@cli.group()
def inspire():
    """INSPIRE utils to update publication information."""
```
then change `@cli.command()` to `@inspire.command()` and add an `inspire` section to `docs/cli.rst`.

Reply:
Grouped and added ```update_all_records_info()```, which calls the ```since``` function with argument 1899-01-01. I checked that [inspire records updated since 1899-01-01](https://inspirehep.net/literature?sort=mostrecent&size=10&page=1&q=external_system_identifiers.schema%3AHEPData%20and%20legacy_version%3A18990101%2B) does indeed get all 9062 records.